src/hipscat_import/catalog/file_readers.py

```diff
@@ -84,6 +84,9 @@ def provenance_info(self) -> dict:
         Returns:
             dictionary with all argument_name -> argument_value as key -> value pairs.
         """
+        all_args = vars(self)
+        if "kwargs" in all_args and "storage_options" in all_args["kwargs"]:
```

Comment:
Can you add a quick comment as to why you are redacting that information?  Something like "We redact the storage_options because it can contain the user keys.